documentation/install_from_packages.markdown

```diff
@@ -24,17 +33,19 @@ Puppet Server versions are tested against the following versions of Java:
 
 Some Java versions may work with other Puppet Server versions, but we do not test or support those cases. Community submitted patches for support greater than Java 11 are welcome. Both Java 8 and 11 are considered long-term support versions and are planned to be supported by upstream maintainers until 2022 or later.
 
-## Install Puppet Server from packages
+## Install the Puppet Server package
 
 1.  [Enable the Puppet package repositories][repodocs], if you haven't already done so.
 
-2.  Install the Puppet Server package by running:
+2.  Install the Puppet Server package by running one of the following commands.
+
+Red Hat operating systems: 
 
 ````
 yum install puppetserver
 ````
 
-or
+Debian and Ubuntu: 
 
 ```
 apt-get install puppetserver
```

Comment:
Is one of the following commands for Red Hat and the other for Debian and Ubuntu? 

```
systemctl start puppetserver	
``` 	
or 	
```	
service puppetserver start
```

Reply:
The former is the newer systemd command, but I think at this point all supported OSes support that? I'm not sure if there is a reason to prefer one over the other at this point.

Reply:
Do you think we could just have `systemctl start puppetserver` for the Server 6 docs then?

Reply:
imo, yes